pkg/danmep/ep.go

```diff
@@ -118,42 +121,19 @@ func createContainerIface(ep danmtypes.DanmEp, dnet *danmtypes.DanmNet, device s
   if err != nil {
     return errors.New("cannot set renamed IPVLAN interface to up because:" + err.Error())
   }
-  sendGratArps(ip, ip6, dstPrefix)
+  if ip != "" {
+    err = arping.GratuitousArpOverIfaceByName(addr4, dstPrefix)
+    if err != nil {
+      return errors.New("cannot send gARP because:" + err.Error())
```

Comment:
let's not promote this event from warning to error which in turn will kill the whole Pod
otherwise looking good

Reply:
done